packages/discord.js/src/structures/ThreadMember.js

```diff
@@ -102,6 +102,7 @@ class ThreadMember extends Base {
   /**
    * Removes this member from the thread.
    * @param {string} [reason] Reason for removing the member
+   * <warn>This parameter is **deprecated**. Reasons cannot be used.</warn>
```

Comment:
Runtime deprecate?

Reply:
Should be done with 43eb7fed7cce3948fcb0ab7cca7bd3c4e8cad790.